app/subscriber/src/features/landing/styled/Landing.tsx

```diff
@@ -23,7 +23,10 @@ export const Landing = styled(Col)`
   .right-panel {
     .commentary-box {
       .headline {
-        max-width: 30em;
+        flex-grow: 1;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
Do the ellipsis appear when the space is limited?  Or does it just hide the text?

Reply:
Ellipsis hides the text if there isn't enough space to show the whole headline content.

Reply:
Flexbox has issues and requires specific css to accomplish this.  Have you confirmed the ellipsis are displayed?